Makefile

```diff
@@ -35,14 +35,18 @@ ifdef GCE_PD_CSI_DEBUG
 	GCFLAGS="all=-N -l"
 endif
 
+ARCH ?= amd64
+
 all: gce-pd-driver gce-pd-driver-windows
 gce-pd-driver: require-GCE_PD_CSI_STAGING_VERSION
 	mkdir -p bin
-	go build -mod=vendor -gcflags=$(GCFLAGS) -ldflags "-X main.version=$(STAGINGVERSION)" -o bin/${DRIVERBINARY} ./cmd/gce-pd-csi-driver/
+	GOARCH=$(ARCH) go build -mod=vendor -gcflags=$(GCFLAGS) -ldflags "-X main.version=$(STAGINGVERSION)" -o bin/${DRIVERBINARY} ./cmd/gce-pd-csi-driver/
 
 gce-pd-driver-windows: require-GCE_PD_CSI_STAGING_VERSION
+ifeq ($(ARCH), amd64)
```

Comment:
Can you add an else block that emits a warning?

```
ifeq ($(ARCH), amd64)
  ...
else
	echo "Warning: gcp-pd-driver-windows only supports ARCH=amd64."
endif
```
And might as well set GOARCH for windows too. Right now we're just building for whatever the default host architecture is.

Reply:
done